ui/package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-react": "^7.34.2",
     "eslint-plugin-react-hooks": "^4.6.2",
     "happy-dom": "^14.12.0",
+    "prettier": "3.3.3",
```

Comment:
Since we can use prettier directly, can the `"eslint-plugin-prettier": "^5.1.3",` devDependency be removed? I couldn't figure out how to use that one btw.

Reply:
yes, in that case you can remove it

Reply:
turns out it might still be needed for eslint